.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -359,11 +359,11 @@ protected ParseResult FracLikeNumberParse(ExtractResult extResult)
 
                 var smallValue = char.IsDigit(numerator[0]) ?
                     GetDigitalValue(numerator, 1) :
-                    GetIntValue(GetMatches(numerator));
+                    GetIntValue(Utilities.RegExpUtility.GetMatches(this.TextNumberRegex, numerator)); // ACAACA

```

Comment:
Is the comment `// ACAACA` use for something?

Reply:
It was only to remember to return to that line when looking for "ACAACA" in the code, but I forgot to delete it. I already solved it, thanks for the feedback.